hw3/01-characters.js

```diff
@@ -1,4 +1,51 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+const genCharacterElements = function genCharacterElements(data) {
+  const sectionElement = document.querySelector('section');
+  sectionElement.classList.add('characters');
+  data.forEach((character) => {
+    const { fullName, title, imageUrl } = character;
+
+    const characterDiv = document.createElement('div');
+    characterDiv.classList.add('character');
+
+    const imageDiv = document.createElement('div');
+    imageDiv.classList.add('image');
+
+    const imgElement = document.createElement('img');
+    imgElement.setAttribute('src', imageUrl);
+    imgElement.setAttribute('alt', `${fullName}, ${title}`);
+    imgElement.setAttribute('width', 225);
+    imgElement.setAttribute('height', 225);
+
+    const nameAndTitleDiv = document.createElement('div');
+    const fullNameElement = document.createElement('h3');
```

Comment:
Consider using the figure and figcaption elements, this exercise would be a good use case for them.